ush/glwu/df_preprocessing.py

```diff
@@ -300,12 +285,18 @@ def get_preprocessed_data(logger, stats_dir, prune_dir, output_base_template,
     )
     df = create_df(
         logger, stats_dir, pruned_data_dir, line_type, date_range, model_list,
-        met_version, clear_prune_dir
+        met_version, clear_prune_dir, verif_type, fcst_var_names, obs_var_names,
+        interp, domain, date_type, date_hours
     )
+    if df is not None and check_empty(df, logger, 'get_preprocessed_data'):
+        df = None
+    return df
+def run_filters(df, logger, verif_type, fcst_var_names, obs_var_names,
+                    interp, domain, date_type, date_range, date_hours):
     df = filter_by_level_type(df, logger, verif_type)
     df = filter_by_var_name(df, logger, fcst_var_names, obs_var_names)
     df = filter_by_interp(df, logger, interp)
-    df = filter_by_obtype(df, logger, obtype)
+#    df = filter_by_obtype(df, logger, obtype)
     df = filter_by_domain(df, logger, domain)
```

Comment:
@SamiraArdani-NOAA Only thing that I can see is that we can remove the commented out line 299 here.  Other than that, we should be OK.  But please remove the commented out line before I approve.

@AliciaBentley-NOAA